docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -645,6 +645,13 @@ prefer to exclude files, like for other cops, add `--auto-gen-only-exclude`
 when running with `--auto-gen-config`. It will still change the maximum if the
 number of excluded files is higher than the exclude limit.
 
+Cop that has `EnforcedStyle` as a configurable option when running with
```

Comment:
I think you can try to polish the grammar in this paragraph a bit, as it reads kind of strange in English. Seems to me this long sentence should be at least 3 sentences.

Reply:
Thank you for review. I have revised the text. How do you like it?